wsmaster/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/jpa/JpaRecipeDao.java

```diff
@@ -101,33 +104,74 @@ public RecipeImpl getById(String id) throws NotFoundException, ServerException {
     }
   }
 
+  /**
+   * Translated query should look like:
+   *
+   * <pre>
+   * SELECT recipe.ID,
+   *        recipe.CREATOR,
+   *        recipe.DESCRIPTION,
+   *        recipe.NAME,
+   *        recipe.SCRIPT,
+   *        recipe.TYPE
+   * FROM {oj RECIPEPERMISSIONS permission
```

Comment:
IWhat does `{oj` mean here?

Reply:
Don;t know. Just copied from [original|https://github.com/codenvy/codenvy/blob/master/wsmaster/codenvy-hosted-api-machine/src/main/java/com/codenvy/api/machine/server/jpa/OnPremisesJpaRecipeDao.java#L49]

Reply:
@mshaposhnik you can remove this. It's was added only 
for clarifying of the resulting query.